API/NoDocs/Microsoft.Extensions.Diagnostics.ResourceMonitoring/NullUtilizationExtensions.cs

```diff
@@ -0,0 +1,8 @@
+// Assembly 'Microsoft.Extensions.Diagnostics.ResourceMonitoring'
+
+namespace Microsoft.Extensions.Diagnostics.ResourceMonitoring;
+
+public static class NullUtilizationExtensions
+{
+    public static IResourceMonitorBuilder AddNullProvider(this IResourceMonitorBuilder builder);
```

Comment:
What's the use case for this outside of tests?

Reply:
Primarily for tests.

Reply:
This API has been removed.